rpc/subscriptions_test.go

```diff
@@ -0,0 +1,338 @@
+package rpc
```

Comment:
Usualy tests go to `rpc_test` pkg. Can you add a comment with a reason here?

Reply:
Yes but we don't have to follow it if its is difficult to use components which make network/db operations. Therefore, I use `package rpc` for mocking which allows me to easily make particular scenarios on events possible.